definitions/variable/energy/energy-prices.yaml

```
@@ -0,0 +1,112 @@
+# List of variables related to energy prices
+
+# Primary energy prices:
+- Price|Primary Energy|Biomass:
+    description: Biomass producer price
+    unit: [EUR_2020/GJ, USD_2010/GJ]
+    weight: Primary Energy|Biomass
+
+- Price|Primary Energy|Coal:
+    description: Coal price at the primary level (i.e. the spot price at the global or
+      regional market)
+    unit: [EUR_2020/GJ, USD_2010/GJ]
+    weight: Primary Energy|Coal
+
+- Price|Primary Energy|Gas:
+    description: Natural gas price at the primary level (i.e. the spot price at
+      the global or regional market)
+    unit: [EUR_2020/GJ, USD_2010/GJ]
+    weight: Primary Energy|Gas
+
+- Price|Primary Energy|Oil:
+    description: Crude oil price at the primary level (i.e. the spot price at
+                the global or regional market)
+    unit: [EUR_2020/GJ, USD_2010/GJ]
+    weight: Primary Energy|Oil
+
+# Secondary energy prices:
+- Price|Secondary Energy|{Final Energy carrier}:
+    description: price at the secondary level of {Final Energy carrier}, i.e. for large scale
+                consumers (e.g. aluminum production, gas power plant)
+    notes: Prices should include the effect of carbon prices.
+    unit: [EUR_2020/GJ, USD_2010/GJ]
+    weight: Secondary Energy|{Final Energy carrier}
+
+# Final energy prices:
+- Price|Final Energy|Industry|{Final Energy carrier}:
+    description: price at the final level in the industrial sector of {Final Energy carrier}. 
+    notes: Prices should include the effect of carbon prices and all other taxes.
+    unit: [EUR_2020/GJ, USD_2010/GJ, EUR_2020/kWh]
+    weight: Final Energy|Industry|{Final Energy carrier}
+
+- Price|Final Energy|Transportation|{Final Energy carrier}:
+    description: price at the final level in the transport sector of {Final Energy carrier}. 
+    notes: Prices should include the effect of carbon prices and all other taxes.
+    unit: [EUR_2020/GJ, USD_2010/GJ, EUR_2020/kWh]
+    weight: Final Energy|Industry|{Final Energy carrier}
+
+- Price|Final Energy|Residential and Commercial|{Final Energy carrier}:
+    description: price at the final level in the residential and commercial sector of {Final Energy carrier}. 
+    notes: Prices should include the effect of carbon prices and all other taxes.
+    unit: [EUR_2020/GJ, USD_2010/GJ, EUR_2020/kWh]
+    weight: Final Energy|Industry|{Final Energy carrier}
```

This seems to be a copy-paste error
```suggestion
    weight: Final Energy|Residential and Commercial|{Final Energy carrier}
```